### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by mrgzi** [RibirX/Ribir#541] [General]
Do you have a discord server or would you like to create new one to create a community?

**Comment by M-Adoo** [comment]
Thanks! Today, I plan to set up a Discord server.

**Comment by M-Adoo** [comment]
@mrgzi I've set up a Discord server for our community. Feel free to join us [here](https://discord.gg/7p3pSrUvmq).

**Reply by mrgzi** [reply]
Its great!
